packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -901,6 +914,10 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
         // Local data store does not have initial summary.
         return {};
     }
+
+    public setRoot(): void {
```

Comment:
Consider adding comments pointing to follow up issues that track next steps (based on earlier comments that this is temporaraly)